### PR TITLE
New Docusaurus based web page!

### PR DESCRIPTION
**Pull request by daniel-otero**
This is mostly an adaptation of the current markdown files, as well as some improvements and small adjustments for the rendering.

The rest is just the node logic, and a GitHub Action to automate the deploy.

Let's see how it works!

### CODE REVIEW

**Review by ruifm — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -139,3 +137,6 @@ get involved:
     your changes, and submit a [Pull
     Request](https://github.com/codee-com/open-catalog/pulls). We'd love to see
     your contributions!
+
+
+[^1]: [Codee](https://www.codee.com)
```
Comment (inaki-amatria) [inline README.md:148]: I wouldn't move this to the end of the page. What do you think?

File: docusaurus.config.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import {themes as prismThemes} from 'prism-react-renderer';
 import type {Config} from '@docusaurus/types';
 import type * as Preset from '@docusaurus/preset-classic';
+import remarkGithubAdmonitionsToDirectives from "remark-github-admonitions-to-directives";
```
Comment (inaki-amatria) [inline docusaurus.config.ts:4]: nit:

```suggestion
import remarkGithubAdmonitionsToDirectives from 'remark-github-admonitions-to-directives';
```

**Review by inaki-amatria — APPROVED** [review]
Looks great!!

File: README.md (hunk 1 above)
Comment (daniel-otero) [reply README.md:148]: This way is closest to how GitHub and docusaurus render it.

Also, footnotes are sopposed to be and the end of the page, right? :laughing:

File: docusaurus.config.ts (hunk 2 above)
Comment (daniel-otero) [reply docusaurus.config.ts]: Done, thanks!

File: README.md (hunk 1 above)
Comment (inaki-amatria) [reply README.md:148]: > This way is closest to how GitHub and docusaurus render it.

I see. Thank you so much for the clarification!

> Also, footnotes are sopposed to be and the end of the page, right? 😆

😆

**Review by alvrogd — COMMENTED** [review]
Looking awesome! I'll update the sidebar with a link to this new website.

**Pull request merged**